target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -46,6 +46,8 @@ avm,fritzdvbc)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan1" "60" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan1" "80" "100"
 	;;
+belkin,f9j1108-v2)
+	;;
```

Comment:
What's that for?

Reply:
Placeholder, but turns out not needed given that there are no LEDs for LAN ports